cli/tests/it/test.rs

```diff
@@ -78,6 +78,22 @@ contract ATest is DSTest {
         )
         .unwrap();
 
+    prj.inner()
+        .add_source(
+            "FailTest.t.sol",
+            r#"
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.10;
+import "./test.sol";
+contract FailTest is DSTest {
+    function testNothing() external {
+        assertTrue(false);
+    }
+}
+   "#,
+        )
+        .unwrap();
+
     cmd.args(["test", "--match-path", "*src/ATest.t.sol"]);
-    cmd.stdout().contains("[PASS]")
+    cmd.stdout().contains("[PASS]") && !cmd.stdout().contains("[FAIL]")
```

Comment:
is this added check not redundant?

Reply:
I updated this so that this confirms the `--match-path` option *only* runs tests that match the path.  
By checking if the output contains a "[PASS]" and also does not contain a "[FAIL]" we make sure that:
1. `ATest.sol` runs correctly (I guess this is redundant because that gets tested in the other tests above)
2. `FailTest.sol` is not run. If it were to be run (implying the path matching has failed) then the output will contain "[FAIL]"